concordium-node/Cargo.toml

```diff
@@ -51,7 +51,7 @@ noiseexplorer_xx = { path = "../deps/external/noiseexplorer_xx" }
 bytesize = "1.0"
 crossbeam-channel = "0.5"
 semver = { version = "1", features = ["serde"] }
-flatbuffers = { version = "2" }
+flatbuffers = { version = "2.1.2" }
```

Comment:
Settled on flatbuffers crate version 2.1.2 since any newer version causes a lot of build errors. Should I look into upgrading it to the latest version or is 2.1.2 fine for now?

Also, I could only find the following security advisories, they both seem to be fixed in >= 2.0.0, so am I overlooking the security advisory we are upgrading to resolve?

https://rustsec.org/advisories/RUSTSEC-2020-0009.htm
https://rustsec.org/advisories/RUSTSEC-2019-0028.html

Reply:
Yes.

And that is why we **also** need to update the flatbuffer compiler. The actual compiler used to generate code from the flatbuffers description.

This is installed on the system, and will be the thing that needs to be updated in the build jobs as well.